src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```

Comment:
Missing header comments. Try writing the comments for a new class before adding any data member or method in it.